tests/core/simulation_control_05.output

```diff
@@ -0,0 +1,36 @@
+
+DEAL::*************************************************
+DEAL::Constant time stepping - constant output
+DEAL::*************************************************
+DEAL::Iteration : 0    Time : 0.00000
+DEAL::Iteration : 1    Time : 0.400000
+DEAL::Iteration : 2    Time : 1.00000
+DEAL::Iteration : 3    Time : 2.00000
+DEAL::Iteration : 4    Time : 3.00000
+DEAL::Iteration : 5    Time : 4.00000
+DEAL::*************************************************
+DEAL::Constant time stepping - constant output
+DEAL::*************************************************
+DEAL::Iteration : 0    Time : 0.00000
+DEAL::Iteration : 1    Time : 0.400000
+DEAL::Iteration : 2    Time : 0.800000
+DEAL::Iteration : 3    Time : 1.00000
+DEAL::Iteration : 4    Time : 2.00000
+DEAL::Iteration : 5    Time : 3.00000
+DEAL::Iteration : 6    Time : 4.00000
+DEAL::*************************************************
+DEAL::Constant time stepping - constant output
+DEAL::*************************************************
+DEAL::Iteration : 0    Time : 0.00000
+DEAL::Iteration : 1    Time : 1.00000
+DEAL::Iteration : 2    Time : 2.00000
+DEAL::Iteration : 3    Time : 3.00000
+DEAL::Iteration : 4    Time : 4.00000
+DEAL::*************************************************
+DEAL::Constant time stepping - constant output
+DEAL::*************************************************
+DEAL::Iteration : 0    Time : 0.00000
+DEAL::Iteration : 1    Time : 1.00000
+DEAL::Iteration : 2    Time : 2.00000
+DEAL::Iteration : 3    Time : 3.00000
+DEAL::Iteration : 4    Time : 4.00000
```

Comment:
This output will not help you debug the test if it crashes. You should have it print something worthwhile (like e.g the method used for the assembly)